app/code/Magento/Review/Block/Adminhtml/Edit/Form.php

```diff
@@ -85,7 +85,8 @@ protected function _prepareForm()
                         [
                             'id' => $this->getRequest()->getParam('id'),
                             'ret' => $this->_coreRegistry->registry('ret'),
-                            'productId' => $this->getRequest()->getParam('productId')
+                            'productId' => $this->getRequest()->getParam('productId'),
+                            'customerId' => $this->getRequest()->getParam('customerId')
```

Comment:
@ravi-chandra3197 here data must not be added to array when it is empty (current code produces lines like ` 'productId' => null`).